luigi_pipeline/seqr_sv_loading.py

```diff
@@ -37,6 +37,13 @@ class SeqrSVVariantMTTask(SeqrVCFToVariantMTTask):
     # NB: electing not to override import_vcf here eventhough the inherited args are slightly different
     # than from the old pipeline.
 
+    def import_dataset(self):
+        mt = self.import_vcf()
+        return mt.key_rows_by('rsid')
+
+    def annotate_old_and_split_multi_hts(self, mt):
```

Comment:
`hl.split_multi_hts` isn't called in the old pipeline.

Reply:
might be worth either a) renaming this in the base class to something more generic and/pr b) adding a comment here to explain that annotating old and solitting multi are not needed for SVs